lib/wallet/api/http/Cardano/Wallet/Api/Types/Transaction.hs

```diff
@@ -8,6 +8,7 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE StrictData #-}
 {-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
```

Comment:
Not sure which change in this file requires the `TypeFamilies` extension.

Reply:
Hi @Unisay: the use of `intCast` requires `TypeFamilies`, and the `Coin.{from,to}Quantity` functions both use `intCast`.

Reply:
Do you mean that `intCast` used inside the `Coin.fromQuantity` propagates this requirement to add `TypeFamilies` to all the call sites?

Reply:
I suspect that `IsIntSubType Natural i ~ 'True` this constraint causes a `TypeFamilies`

Reply:
> Do you mean that intCast used inside the Coin.fromQuantity propagates this requirement to add TypeFamilies to all the call sites?
Yes
> I suspect that IsIntSubType Natural i ~ 'True this constraint causes a TypeFamilies
Indeed!